html/index.html

```diff
@@ -76,6 +76,289 @@ <h1>
+          css
+      </li>
+      <li>
+          javascript
+      </li>
+  </ul>
+</div>
+<div class="Projects">
+  <button type="button">
+      See Project
+  </button>
+</div>
+
+
+      </section>
+
+
+      <section class="About-me">
+        <div class="title">
+          <h1>
+            About Myself
+          </h1>
+      
+          <p>
+            Hello I’m a software developer! I can help you build a product , 
+            feature or website Look through some of my work and experience!
+             If you like what you see and have a project you need coded, don’t hestiate to contact me.
+          </p>
+          <p id="About-connect">LET'S CONNECT </p>
+    
+        </div>
+        <div>
+          <ul class="social-media" id="About-social">
+              <li>
+                  <a href="https://twitter.com/Mbonabucya12">
+                    <img src="../Image/Vector-twitter.svg" alt="Twitter icon" />
+                  </a>
+              </li>
+              <li>
+                  <a href="https://www.linkedin.com/in/james-mbonabucya-88666910b/">
+                    <img src="../Image/vector-linkedin.svg" alt="Linkedin icon" />
+                  </a>
+              </li>
+              <li>
+                  <a href="https://www.gmail.com">
+                    <img src="../Image/Vector-Medium.svg" alt="Medium icon" />
+                  </a>
+
+              </li>
+              <li>
+                  <a href="https://github.com/mbonabucya">
+                    <img src="../Image/Vector-github.svg" alt="Github icon" />
+                  </a>
+              </li>
+              <li>
+                <a href="https://github.com/mbonabucya">
+                  <img src="../Image/Vector-peace.svg" alt=" peace icon" />
+                </a>
+            </li>
+          </ul>
+        </div>
+         <div class="resume">
+             <button type="button">Get my resume</button>
+         </div>
+           <div class="languages">
+
+               <div class="front-end-languages-header">
+                   <h1>Languages</h1> 
+                   <img src="../Image/dropdown.png" alt=" drop down icon" />
+               </div>
+               <div class="front-end">
+                   <img src="../Image/Ellipse-js.png" alt="js">
+                   <p>JavaScript</p>
+               </div>
+               <div class="front-end">
+                <img src="../Image/Ellipse-html.png" alt="js">
+                <p>Html</p>
+            </div>
+
+            <div class="front-end">
+                <img src="../Image/Ellipse-css.png" alt="js">
+                <p>CSS</p>
+            </div>
+            <div class="frame-work">
+                <h1>Frameworks</h1>
+                <img src="../Image/right-arrow.png" alt="">
+            </div>
+            <div class="skills">
+                <h1>Skills</h1>
+                <img src="../Image/right-arrow.png" alt="">
+            </div>
+
+
+           </div>
```

Comment:
How about wrapping all the technology categories in a `<ul>` tag and each category in a `<li>` tag